### ISSUE TITLE
Does not work with EIN buffers.

### ISSUE THREAD

**Issue by lgmoneda** [anler/centered-window-mode#27]
## Expected behavior
Center an EIN buffer.

## Actual behavior
It works for other buffers, but not with EIN ones. But i can center it manually using fringes (is that what the mode does?) with ```(fringe-mode '(200 . 200))```.

## Steps to reproduce
- ein:notebooklist-open
- M-x centered-window-mode

## Versions of `centered-window-mode`, Emacs, OS etc.

- `centered-window-mode`: 20170528
- Emacs: 25.2
- OS: Debian

**Comment by anler** [comment]
Hi @lgmoneda by default all buffers starting by `*` (except `*scratch*`) are ignored, is the name of your buffer like that?

**Comment by lgmoneda** [comment]
Indeed, @anler, the EIN buffers all start with a ```*```. Is there a variable to customize it? Thanks for helping!

**Comment by anler** [comment]
@lgmoneda the customization variable is `cwm-ignore-buffer-predicates` which is a list of functions to test if a a given buffer should be ignored, by default it contains only the function `cwm-special-buffer-p`. If you want to activate the mode in all special buffers just add `(setq cwm-ignore-buffer-predicates nil)` to your emac's init, otherwise you can create a new function that uses internally `cwm-special-buffer-p` but also checks that the buffer is not an EIN one.
